.eslintrc

```diff
@@ -13,7 +14,7 @@
         "indent": [2, 2],
         "curly": [2, "multi-line"],
         "no-use-before-define": [2, "nofunc"],
-        "no-unused-vars": [2, "all"],
+        "no-unused-vars": [1, { "args": "none" }],
```

Comment:
I prefer to check for *all* unused vars, that's why this was explicitly set that way.